### PR TITLE
Fix GitHub Actions

### PR DESCRIPTION
**Pull request by tinovyatkin**
Recovery for #785

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -9,17 +9,12 @@ on:
...
-
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js
       uses: actions/setup-node@v1
       with:
-        node-version: ${{ matrix.node-version }}
+        node-version: 14
```
Comment (NotMoni) [inline .github/workflows/lint.yml:18]: might wanna do `latest` here.

**Comment by NotMoni** [comment]
all ci and Github actions now finally seem to work.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (tinovyatkin) [reply .github/workflows/lint.yml:18]: I believe GitHub Action doesn't have `latest` and `lts`

**Comment by tinovyatkin** [comment]
Screenshot 2020-05-22 at 17 31 47

**Comment by tinovyatkin** [comment]
Waiting for Actions to revive...

**Pull request merged**